src/frontend/src/app/login/login.component.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+    margin-top:25px;
+  }
+  
+  .login {
```

Comment:
It's probably minor OCD but it'd be better to have these lines match up by sliding all the `.login` pieces to the left.

Reply:
The html file should not have a html, head, body, etc tags.  This is all in the index.html page out in the root of the project.  Just start with your div.

Reply:
The login page is also not making use of material